hash_practice/exercises.py

```diff
@@ -1,19 +1,45 @@
 
+from typing import OrderedDict
+
+
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(nlog(n))

```

Comment:
My guess is that you noticed your call to `sorted` and are accounting for that, with `n` referring to the number of characters in the strings (as that's what affects how long `sorted` takes).

However, don't forget about the number of strings in the list (let's call that `m`), which will call `sorted` each time for O(m * n * log(n)).

But the good news is that we can actually make a simplifying assumption! Since we know the strings are English words, and English words tend to be limited in length (about 5 letters on average), the effect of `n` is going to be dwarfed by `m`, the number of strings in `strings`. There may be hundreds or thousands of strings in `strings`, after all. With that we can give the time complexity as O(n).